src/test/java/org/opensearch/security/sanity/tests/SingleClusterSanityIT.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2015-2018 _floragunn_ GmbH
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * Portions Copyright OpenSearch Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package org.opensearch.security.sanity.tests;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+@SuppressWarnings("unchecked")
+public class SingleClusterSanityIT extends SecurityRestTestCase {
+
+    private static final String SECURITY_PLUGIN_NAME = "opensearch-security";
+
+    @Test
+    public void testSecurityPluginInstallation() throws Exception {
+        verifyPluginInstallationOnAllNodes();
+    }
+
+    private void verifyPluginInstallationOnAllNodes() throws Exception {
```

Comment:
This is nice start. 
Do you plan on writing/migrating tests which verify security plugin functionality?

Reply:
yes...that is next step in the evolution of this testing process.

Reply:
Nice, do you have a follow up issue which takes care of it?

Reply:
Yes we track it in this parent issue: https://github.com/opensearch-project/security/issues/1761